main.cpp

```diff
@@ -429,6 +512,8 @@ struct Display
     void displayDisplayInfo(std::string brandName, int year, std::string modelName);
     // set color calibration
     std::string changeColorCalibration(std::string displayPreference = "None");
+    // new member function
+    Display getNewDisplay(int threshold, int startingValue);
```

Comment:
a Display instance will return another Display?
that doesn't make much sense.
This is a member function, so the only way to call it is by using a Display instance...
- [ ] revise